nimble/host/src/ble_hs.c

```diff
@@ -734,6 +734,9 @@ ble_hs_init(void)
     rc = ble_l2cap_init();
     SYSINIT_PANIC_ASSERT(rc == 0);
 
+    rc = ble_gap_init();
```

Comment:
we need to do something with line 752 I guess

Reply:
Also, `ble_gap_init` should be outside `#if NIMBLE_BLE_CONNECT` - it includes code for features without connection

Reply:
fixed